website/static/files/build_orca_reconcile.sql

```diff
@@ -6,88 +6,94 @@ SET search_path TO orca, public;
-    CONSTRAINT PK_reconcile_job PRIMARY KEY(id),
-    CONSTRAINT FK_reconcile_job_status FOREIGN KEY(status_id) REFERENCES reconcile_status(id)
+  CREATE TABLE IF NOT EXISTS reconcile_jobs  ( 
+    id                     	int8 NOT NULL
+  , orca_archive_location  	text NOT NULL
+  , status_id              	int2 NOT NULL
+  , inventory_creation_time	timestamp with time zone NOT NULL
+  , start_time             	timestamp with time zone NOT NULL
+  , last_update            	timestamp with time zone NOT NULL
+  , end_time               	timestamp with time zone NULL
+  , error_message          	text NULL
+  , CONSTRAINT PK_reconcile_job PRIMARY KEY(id)
+  , CONSTRAINT FK_reconcile_job_status FOREIGN KEY(status_id) REFERENCES reconcile_status(id)
   );
   COMMENT ON TABLE reconcile_jobs IS 'Manages internal reconciliation job information.';
   COMMENT ON COLUMN reconcile_jobs.id IS 'Job ID unique to each internal reconciliation job.';
-  COMMENT ON COLUMN reconcile_jobs.orca_archive_location IS 'ORCA S3 Glacier bucket the file resides in.';
+  COMMENT ON COLUMN reconcile_jobs.orca_archive_location IS 'ORCA S3 Glacier bucket the reconciliation targets.';
   COMMENT ON COLUMN reconcile_jobs.status_id IS 'Current status of the job.';
-  COMMENT ON COLUMN reconcile_jobs.inventory_creation_time IS 'Time from the s3 inventory manifest ';
+  COMMENT ON COLUMN reconcile_jobs.inventory_creation_time IS 'Inventory report creation time from the s3 manifest.';
   COMMENT ON COLUMN reconcile_jobs.start_time IS 'Date and time the internal reconcile job started.';
   COMMENT ON COLUMN reconcile_jobs.last_update IS 'Date and time the job status was last updated.';
   COMMENT ON COLUMN reconcile_jobs.end_time IS 'Time the job completed and wrote the report information.';
   COMMENT ON COLUMN reconcile_jobs.error_message IS 'Critical error the job ran into that prevented it from finishing.';
 
 
-  CREATE TABLE reconcile_s3_objects  ( 
-    job_id               	int8 NOT NULL,
-    orca_archive_location	text NOT NULL,
-    key_path             	text NOT NULL,
-    etag                 	text NOT NULL,
-    last_update          	timestamp with time zone NOT NULL,
-    size_in_bytes        	int8 NOT NULL,
-    storage_class        	text NOT NULL,
-    delete_marker        	bool NOT NULL,
-    CONSTRAINT FK_reconcile_jobs_s3_objects FOREIGN KEY(job_id) REFERENCES reconcile_jobs(id)
-    ) PARTITION BY RANGE (orca_archive_location);
+  CREATE TABLE IF NOT EXISTS reconcile_s3_objects  ( 
+    job_id               	int8 NOT NULL
+  , orca_archive_location	text NOT NULL
+  , key_path             	text NOT NULL
+  , etag                 	text NOT NULL
+  , last_update          	timestamp with time zone NOT NULL
+  , size_in_bytes        	int8 NOT NULL
+  , storage_class        	text NOT NULL
+  , delete_marker        	bool NOT NULL
+  , CONSTRAINT FK_reconcile_jobs_s3_objects FOREIGN KEY(job_id) REFERENCES reconcile_jobs(id)
+    ) PARTITION BY LIST (orca_archive_location);
```

Comment:
Still missing Primary Key, you will need this key of `orca_archive_location` and `key_path` in order to create your partition by list.

Reply:
[From our previous conversations, we will create the index after the partitioned data is ingested. Admittedly, I do not know what that looks like.
Also, this is somehow three different threads now.](https://github.com/nasa/cumulus-orca/pull/230#discussion_r789877173)

Reply:
Switching to new plans to use Truncate.